remote-trait-object/src/ipc/multiplex.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use std::sync::mpsc::{channel, Receiver};
+use std::thread;
+
+pub struct MultiplexResult {
```

Comment:
How about `MultiplexEntries`?

Reply:
Having `Multiplexer` in `MultiplexEntries` seems strange.
Could you tell me why `MultiplexEntries` is a better name?

Reply:
Ok. Let's keep `MultiplexResult`.